tests/test_operators.py

```diff
@@ -775,12 +775,12 @@ async def test_folder_object_remove_fails(self):
         with self.assertRaises(HTTPBadRequest):
             await operator.remove_object(self.test_folder, "study", self.accession_id)
 
-    async def test_check_folder_exists_fails(self):
+    async def testcheck_folder_exists_fails(self):
```

Comment:
Does this need an underscore `_` to `testcheck` -> `test_check` to be discovered by the test suite?

Reply:
will work without it, but reverting to keep naming convention.